docker/hive/start-hive.sh

```diff
@@ -7,6 +7,16 @@ METASTORE_PORT=${METASTORE_PORT:=9083}
 DEFAULT_FS=${DEFAULT_FS:=file:///}
 DB_TYPE=${DB_TYPE:=mysql}
 
+cat >${HIVE_HOME}/conf/hive-log4j.properties <<EOL
+log4j.rootLogger=INFO, CONSOLE
+
+# Disable excessive reflection warnings - KAFKA-5229
```

Comment:
Add a file to the docker log4j.json then copy it here if you got use_json_logger env variable

Reply:
@lyogev i dont understand